drivers/ethernet/phy/Kconfig

```diff
@@ -40,6 +40,12 @@ config PHY_ADIN2111
 	help
 	  Enable ADIN2111 PHY driver.
 
+config PHY_DM8806
+	bool "DM8806 PHY driver"
+	default y
```

Comment:
You need to add `depends on DT_HAS_DAVICOM_DM8806_PHY_ENABLED`

Reply:
Agree - dependency added